Esri Open Source Community Code of Conduct

ChrisFrenchPDX opened the pull request:
Initial draft for an Esri Open Source Community Code of Conduct

mikewilburn commented on COMMUNITY_CODE_OF_CONDUCT.md:
```diff
@@ -0,0 +1,40 @@
+- Report inappropriate behavior
+- Contributors must have the legal right to contribute source code and content under the Apache 2.0 license
+
+## Inappropriate behavior—harassment, aggression, and intimidation
```
I don't think we should limit this section heading to "harassment, aggression, and intimidation" as your second paragraph points out. I think simply "Inappropriate behavior" is fine.

mikewilburn commented on COMMUNITY_CODE_OF_CONDUCT.md:
```diff
@@ -0,0 +1,40 @@
+
+- Inappropriate behavior will not be tolerated. If asked to stop behaving inappropriately, you should comply immediately
+- Esri does not tolerate harassment or personal threats
+- Violations of this Community Code of Conduct may result in disqualifications in Esri Events, including conferences. [See the Esri Code of Conduct for Events and Conferences](https://www.esri.com/en-us/about/events/code-of-conduct).
```
Some of these aren't consequences and just restate that there _will_ be consequences. I propose that we add this as a course of action:
https://help.github.com/articles/blocking-a-user-from-your-organization/

jf990 commented:
Can we consider naming the file CODE_OF_CONDUCT.md ? That seems to be the npm/OSS standard. Someone even has a [CoC generator](https://github.com/WeAllJS/weallbehave)

ChrisFrenchPDX commented:
Ok, I've made a few changes and edits based on everyone's suggestions:

- @jf990 I deleted the `COMMUNITY_CODE_OF_CONDUCT.md` file and moved the contents to `CODE_OF_CONDUCT.md`
- @mikewilburn  and @jgravois I added a version/data to the doc, moved some the contributing info the the `README.md` file, clarified consequences for violations, and restructured the file re the suggestions

jgravois commented:
@ChrisFrenchPDX i have a few more copy editing nitpicks. can i just push them up in a commit of my own?

ChrisFrenchPDX commented:
@jgravois Sure!

ChrisFrenchPDX commented:
Ok, cool. The changes look good to me - I just changed a few punctuation typos.

mikewilburn commented:
This is looking real solid to me. Happy with the latest changes!
